README.md

```diff
@@ -157,7 +158,7 @@ $ kubectl delete catalogsourceconfig installed-upstream-community-operators -n m
 
 Follow the steps [here](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) to upload operator artifacts to `quay.io`.
 
-Once your operator artifact is pushed to `quay.io` you can use an `OperatorSource` to add your operator offering to Marketplace. An example `OperatorSource` is provided [here](deploy/examples/upstream.operatorsource.cr.yaml).
+Once your operator artifact is pushed to `quay.io` you can use an `OperatorSource` to add your operator offering to Marketplace. An example `OperatorSource` is provided [here](deploy/upstream/examples/upstream.operatorsource.cr.yaml).
```

Comment:
This change is unneeded since the previous path was correct

Reply:
It has been moved to deploy/upstream as part of this PR as per @kevinrizza's request.